functions/test/tips/scopes_test.rb

```diff
@@ -22,7 +22,7 @@
       request = make_get_request "http://example.com:8080/"
       response = call_http "tips_scopes", request
       assert_equal 200, response.status
-      assert_equal "instance: heavy computation result; function: light computation result", response.body.join
+      assert_match (/instance: .*; function: .*/, response.body.join)
```

Comment:
Oh wait, sorry, no space between `assert_match` and the opening paren.

Reply:
My bad! Fixed. Please recheck.